frontend/src/lib/components/Subscriptions/SubscriptionsModal.tsx

```diff
@@ -10,23 +9,25 @@ import { PayGatePage } from '../PayGatePage/PayGatePage'
 import { AvailableFeature } from '~/types'
 import { userLogic } from 'scenes/userLogic'
 import { Spinner } from '../Spinner/Spinner'
+import { LemonModal } from '../LemonModal'
 
 export interface SubscriptionsModalProps extends SubscriptionBaseProps {
     visible: boolean
     closeModal: () => void
     subscriptionId: number | 'new' | null
+    inline?: boolean
 }
 
-export function Subscriptions(props: SubscriptionsModalProps): JSX.Element {
-    const { closeModal, dashboardId, insightShortId, subscriptionId } = props
+export function SubscriptionsModal(props: SubscriptionsModalProps): JSX.Element {
+    const { closeModal, dashboardId, insightShortId, subscriptionId, visible, inline } = props
     const { push } = useActions(router)
     const { hasAvailableFeature, userLoading } = useValues(userLogic)
 
     if (userLoading) {
         return <Spinner />
     }
     return (
-        <>
+        <LemonModal onClose={closeModal} isOpen={visible} width={600} simple title="" inline={inline}>
```

Comment:
`isOpen` nicer language than `visible` for a modal

but the repeated `isOpen={visible}` is yucky... rename the variables?

Reply:
Yup was thinking the same so I should do it now otherwise I'll forget